docker/README.md

```diff
@@ -83,6 +83,10 @@ The following environment variables are supported:
 
   The hostname for the puppetserver instance. This determines where to request certificates from. Defaults to 'puppet'.
 
+- `PUPPETSERVER_PORT`
```

Comment:
Added this due to a similar PR from @mcanevet at https://github.com/puppetlabs/puppetserver/pull/2060